Create vsp-release-plan.md (#12493)
department-of-veterans-affairs/va.gov-team: .github/ISSUE_TEMPLATE/vsp-release-plan.md

> name: VSP Initiative Release Plan Template (Draft)

Reviewer: ```suggestion
name: VSP Initiative Release Plan (Draft)
```

> # Release Plan Template [Draft]

Reviewer: this needs a stanza at the top with template defaults

```
---
name: VSP Initiative Release Plan Template (Draft)
about: [... some description here ...]
title: "[Initiative-Name]: Release Plan"
---
```

and are there any labels we want by default?

> # Release Plan Template [Draft]

Reviewer: I think you can just delete this first line if we have the issue title already

> This Release Plan Template is intended to be used to help ensure your new VSP product or feature is ready for launch and will deliver the desired user outcomes. In Phase 1, you will test your product to learn how it'll actually be used, what problems it might create, and then fix/adjust if necesssary prior to going live. In Phase 2, you will define the launch communications plan and post-launch KPIs. 
> 
> You do not need to complete a release plan for a product or feature update that does not:
> 
> - Add substantial new functionality to the Platform.
> - Impact user flows through tasks.
> 
> For your product or feature, make a copy of this template and check it into products/platform/<product-name>.

Reviewer: deleting a couple things:
- some of the "when to use this" text (assumption: if you're here, you've _already_ been directed to use a release plan)
- directions about copying to a certain directory (nope, moving to this github issue instead)

adding some things:
- italicized "Guidance" section
- directions about filling it out

```suggestion
## Guidance

_This Release Plan Template is intended to help ensure your new VSP product or feature is ready for launch and will deliver the desired user outcomes. In Phase 1, you will test your product to learn how it'll actually be used, what problems it might create, and then fix/adjust if necessary prior to going live. In Phase 2, you will define the launch communications plan and post-launch KPIs.)

_Fill out the **Planning** sections to start, then the **Results** sections as you complete each phase of testing._

---
```

> 

Reviewer: one more question on guidance: when do we think users can/should _close_ this ticket, and how can we walk them through those expectations?

I'd normally expect "launch" for a release plan template, but we also have a lot of "post-launch" information we ask people for in this template. 🤔 I'm open to ideas here

Author: I pulled the post-launch info from the VFS release plan template. They do post-launch reviews which I don't think we do (at least not today). Perhaps we can mark that section as Optional for users who want to document post-launch KPIs and say you can close the ticket after launch.

Author: I created a separate "Phase" for the post-launch items and indicated it's optional. Updated the Guidance instructions to say users can close the ticket at launch unless they want to document post-launch results.

## Phase I: User Acceptance Testing
*Test real-world user scenarios to ensure your product or feature performs as expected.*

### Planning:

- Desired date range or test duration: 
- Desired number of users: 
- The group(s) from which you'll recruit test users:
  - [ ] VFS
  - [ ] VSP
  - [ ] DEPO

> - How you'll recuit test users:

Reviewer: ```suggestion
- How you'll recruit test users:
```

- How you'll give test users access to the product: 
- What metrics-based "success criteria" will you look at before launch? [use your KPIs to help guide this.]
- How you'll measure your "success criteria": 

### Results:

- Number of users: 	

- Results for your "success criteria":

- Number of bugs identified / fixed: x/x

- Any UX changes necessary based on the logs, user feedback, or VA challenges? yes/no

  - If yes, what:   

## Phase II: Go Live!

> ### Launch communications:

Reviewer: What are we expecting people to do with these checkboxes / bullet points?

How does this overlap with what's already covered in the initiative development checklist?

- [ ] Product documentation
  - Product README
  - User Guide 
- [ ] Release announcements
  - Newsletter article - collaborate with the Service Design team
  - Team of Teams and other VFS/VSP meetings

### Planning:

- Desired launch date: 

> - Post-launch KPI 1:

Reviewer: are KPIs always required? does this make sense for VSP products?
(can you give me examples of what this would look like for, say, VSP Newsletter, or another product you're familiar with?)

Reviewer: bump on this question

Author: I think we are required to set KPIs as part of the product outline. For the Newsletter, the KPIs are:
- successful email delivery rate
- email open rate
- click rate per unique opens

Reviewer: sure, I understand we set KPIs for product success --- just wondering if KPIs are necessary as part of the release plan also for VSP?

i.e. for VSP Newsletter, did you look at any of those KPIs as success criteria before launch?

name: VSP Initiative Release Plan Template (Draft)

about: Release plan template for VSP products.

title: [Initiative-Name]: Release Plan

---

## Guidance

> _This Release Plan Template is intended to help ensure your new VSP product or feature is ready for launch and will deliver the desired user outcomes. In Phase 1, you will test your product to learn how it'll actually be used, what problems it might create, and then fix/adjust if necessary prior to going live. In Phase 2, you will define the launch communications plan and post-launch KPIs. You can close this ticket once you have completed Phase 2. Phase 3 is optional - you may use it to document post-launch results. 

Reviewer: ```suggestion
_This Release Plan Template is intended to help ensure your new VSP product or feature is ready for launch and will deliver the desired user outcomes. In Phase 1, you will test your product to learn how it'll actually be used, what problems it might create, and then fix/adjust if necessary prior to going live. In Phase 2, you will define the launch communications plan and post-launch KPIs. You can close this ticket once you have completed Phase 2. Phase 3 is optional - you may use it to document post-launch results._
```
